dojo/finding/views.py

```diff
@@ -513,7 +516,7 @@ def open_findings(request, pid=None, eid=None, view=None):
 
     found_by = None
     try:
-        found_by = findings.found_by.all().distinct()
+        found_by = findings_filter.found_by.all().distinct()
```

Comment:
why do we work on the findings_filter here instead of findings?

Reply:
because that is what the old code was also doing. findings doesn't have an attribute/property called found_by.